cine_index.css

```diff
@@ -0,0 +1,206 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  margin: auto;
+  font-family: Haettenschweiler, "Arial Narrow Bold", sans-serif;
+  background-color: #242033f1;
+  color: #f2faf2;
+  text-align: center; 
+}
+
+p{
+  margin:0px
+}
+
+.App__header{
+  display:flex;
+  flex-direction: row;
+  flex-wrap:wrap;
+
+  height:130px;
+  background:url(MRNmobile2.png);
+
+  background-repeat: no-repeat;
+  background-position: center;
+
+  text-align:center;
+}
+
+input {
```

Comment:
be careful around applying styling  directly on input tags. there are different input types and some will require different styling approaches. If you styling just text inputs, use a class name and apply it to components you wish to style